zhaquirks/xiaomi/aqara/motion_agl02.py

```diff
@@ -0,0 +1,85 @@
+"""Xiaomi aqara T1 motion sensor device."""
+from zigpy.profiles import zha
+from zigpy.zcl.clusters.general import Identify, Ota
+from zigpy.zcl.clusters.measurement import OccupancySensing
+
+from zhaquirks import Bus
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.xiaomi import (
+    LUMI,
+    BasicCluster,
+    IlluminanceMeasurementCluster,
+    MotionCluster,
+    OccupancyCluster,
+    XiaomiAqaraE1Cluster,
+    XiaomiCustomDevice,
+    XiaomiPowerConfiguration,
+)
+
+XIAOMI_CLUSTER_ID = 0xFCC0
+
+
+class XiaomiManufacturerCluster(XiaomiAqaraE1Cluster):
+    """Xiaomi manufacturer cluster."""
+
+    def _update_attribute(self, attrid, value):
+        super()._update_attribute(attrid, value)
+        if attrid == 274:
+            value = value - 65536
+            self.endpoint.illuminance.illuminance_reported(value)
```

Comment:
So illuminance update is also a trigger for  occupancy?

Reply:
unfortunately.... I'll link you the z2m thread

Reply:
Start here: https://github.com/Koenkk/zigbee2mqtt/issues/5706#issuecomment-971810515

Reply:
What [censored]...